tests/Fixtures/e2e/Composer_Config_Custom_Vendor_Dir/run_tests.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+readonly INFECTION=../../../../bin/infection
+
+rm -rf infection-file.txt
+
+set -e pipefail
+
+if [ "$PHPDBG" = "1" ]
+then
+    phpdbg -qrr $INFECTION
+else
+    php $INFECTION
+fi
+
+diff -w expected-output.txt infection.log
+diff -w expected-file.txt infection-file.txt
```

Comment:
Is it not created by anyone?

Reply:
No, removed

Reply:
This was a fine script, but `expected-file.txt` and `infection-file.txt` were out of band.